instrumentation/opentelemetry-instrumentation-botocore/src/opentelemetry/instrumentation/botocore/extensions/bedrock.py

```diff
@@ -305,11 +380,25 @@ def _get_request_messages(self):
 
                 messages = decoded_body.get("messages", [])
                 if not messages:
-                    # transform old school amazon titan invokeModel api to messages
-                    if input_text := decoded_body.get("inputText"):
-                        messages = [
-                            {"role": "user", "content": [{"text": input_text}]}
-                        ]
+                    model_id = self._call_context.params.get(_MODEL_ID_KEY)
+                    if "amazon.titan" in model_id:
+                        # transform old school amazon titan invokeModel api to messages
```

Comment:
I think we can add a more generic comment on top that says that these are models that does not provide a messages interface

Reply:
Done